clusterloader2/pkg/prometheus/manifests/exporters/kube-state-metrics/0namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
I wouldn't put that into pkg/prometheus.

Can you please move that to:
pkg/kube-state-metrics maybe?

[i.e. pkg/kube-state-metrics/manifests to be more specific]

Reply:
Actually - we put there a bunch of stuff, so maybe that's fine. Feel free to ignore (we can refactor that in the future).

Reply:
Yeah was not sure where to put it, so placed it under exporters, agreed its not ideal.